core/templates/domain/topic_viewer/topic-viewer-backend-api.service.ts

```diff
@@ -35,8 +35,11 @@ export class TopicViewerBackendApiService {
 
   private topicDataDict = null;
   private _fetchTopicData(
-      topicName: string, successCallback: any, errorCallback: any): any {
-    var topicDataUrl = this.urlInterpolation.interpolateUrl(
+      topicName: string,
+      successCallback: (value?: Object | PromiseLike<Object>) => void,
```

Comment:
I don't understand this change.. @bansalnitish is this the right type?

Reply:
@nithusha21 in my other pull requests @bansalnitish has asked me to follow this convention for successCallback and errorCallback

Reply:
Ah ok, I am not very sure I understand the type... But I see it all over the codebase... So I guess this is fine!

-